core/geometry/src/geometry3d/Matrix3d.ts

```diff
@@ -289,7 +289,7 @@ export class Matrix3d implements BeJSONFunctions {
     }
   }
   /** Return a new Matrix3d constructed from contents of the json value. Se `setFromJSON` for layout rules */
-  public static fromJSON(json?: Matrix3dProps): Matrix3d { const result = Matrix3d.createIdentity(); result.setFromJSON(json); return result; }
+  public static fromJSON(json?: Matrix3dProps | Matrix3d): Matrix3d { const result = Matrix3d.createIdentity(); result.setFromJSON(json); return result; }

```

Comment:
This doesn't seem right?

Reply:
My branch was so old, that I somehow picked up a set of unintended changes along the way. I have reverted them now.